Create ISSUE_TEMPLATE.md

jhlegarreta opened the pull request:
Create an `ISSUE_TEMPLATE.md` file to provide a guide when submitting issues to ITK.

hjmjohnson commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,46 @@
+
+### Label
+
+[Choose the label that best fits the issue]
```
Give a list of labels here to choose from.  These labels should match the GITHUB labels for issues.

hjmjohnson replied:
Or at least give a URL of where to find the labels.

jhlegarreta replied:
@hjmjohnson thanks for the review ! This was just a proposal; the question is whether we will effectively be using labels. The VTK folks do have some, and they do use them frequently. If for any reason it is decided that we will not be using labels (despite I'd be for proposing/using them), then adding such a section is not worthwhile.

thewtex replied:
Maybe we can add the *Label* and *Milestones* section later once we have figured out what labels to use and sorted out the milestoning system.

hjmjohnson commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,46 @@
+
+[Choose the label that best fits the issue]
+
+### Milestones
```
Give a URL of where to find the list of Milestones.

jhlegarreta replied:
Same comment as for the labels.

jhlegarreta commented:
An additional comment as for checking the ITK version: we could run `ctest -V -R itkVersionTest -C` but looks like an overkill, and the user may not have built the tests/may have had ITK installed (i.e. not compiled). Inspecting the main `CMakeLists.txt` may be invalid if having ITK installed.

Any thoughts or improvements?

thewtex commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,46 @@
+[Before submitting an issue, please check that your isssue hasn't been already filed]
```
isssue -> issue

jhlegarreta commented:
Done. Commit 7970cf06da4da970da10c0bf75d255639e317e26 removes the `Label` and `Milestone` sections.

I also changed the section name `Platform` for `Environment`. If you think that `Platform` or any other word like `Toolchain` for example fits better, let me know.

I also added a note at the bottom to invite users to ask questions in discourse. If you think it requires rewording, let me know. 

If you think the style of the comments should be changed (e.g. italics instead of or in addition to square brackets), let me know.

thewtex commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,48 @@
+
+[Any additional information, configuration or data that might be necessary to reproduce the issue.]
+
+
```
isses -> issues

jhlegarreta replied:
Fixed.

jhlegarreta commented:
@thewtex Thanks !

I've submitted a topic to gerrit:
http://review.source.kitware.com/#/c/22869/

to further review/merge this. I'm closing this PR then.
